stan/math/prim/prob/cauchy_log.hpp

```diff
@@ -22,9 +22,9 @@ namespace math {
  * @param mu (Sequence of) location(s).
  * @param sigma (Sequence of) scale(s).
  * @return The log of the product of densities.
+ * @tparam T_scale Type of scale.
```

Comment:
[comment]
We want to head toward lower case and no periods for phrases that are not complete sentences.

Reply:
Yes, that's where I'm heading to. I think what happened here is that I started fixing this file, then noticed it was one of the deprecated ones and forgot to undo completely my changes.